Neighborly/VectorList.cs

```diff
@@ -7,13 +7,11 @@ namespace Neighborly;
 /// </summary>
 public class VectorList : IList<Vector>, IDisposable
 {
-    private List<Vector?> _vectorList = new();
-    private readonly List<Guid> _guids = new();
-    public List<Guid> Guids => _guids;
-    private readonly List<string> _onDiskFilePaths = new();
+    private readonly List<Vector?> _vectorList;
     private readonly VectorTags _tags;
     public VectorTags Tags => _tags;
     private readonly int _maxInMemoryCount;
+    private readonly MemoryMappedList _memoryMappedList = new(int.MaxValue * 2L);
```

Comment:
Should this be Int.MaxValue?

Reply:
I used this intentionally, as it's one of the larger files that worked. You can decide this capacity to be whatever you want. 👍🏻